src/chericat.c

```diff
@@ -99,11 +123,26 @@ main(int argc, char *argv[])
 		return 0;
 	}
 
-	sqlite3 *db = NULL;
-
 	static int debug_level = 0;
 	set_print_level(debug_level);
 
+#ifdef HAVE_SIGACTION
+{
+	struct sigaction sa;
+	sa.sa_flags=0;
+	sa.sa_handler = sigterm;
+	sigaction(SIGINT, &sa, NULL);
+	sigaction(SIGTERM, &sa, NULL);
+	sigaction(SIGPIPE, &sa, NULL);
+	sigaction(SIGQUIT, &sa, NULL);
+}
+#else
```

Comment:
As far as I can tell nothing is defining HAVE_SIGACTION so this code block is never used and the signal(3) block below is.

Reply:
I noticed the sigaction(2) is being used elsewhere in cheribsd and thought that I should offer that as an alternative. But yes, it's not defined in chericat and therefore is unused. I have removed it now. Thanks!